packages/x-provider/typedoc.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": ["../../typedoc.base.json"],
   "entryPoints": ["src/index.ts"],
-  "name": "immutablex-client",
+  "name": "x-provider",
```

Comment:
same here. removed a client but added a provider. i don't get the logic.

Reply:
good catch... must have mixed these up

Reply:
Actually, the name is correct for the typedoc configs...